backend/testfiles/execution/stdlib/parser.dark

```diff
@@ -212,6 +212,8 @@ module TextToTextRoundtripping =
     ("type MyFloat = Float" |> roundtripCliScript) = "type MyFloat =\n  Float"
     ("type MyChar = Char" |> roundtripCliScript) = "type MyChar =\n  Char"
     ("type MyString = String" |> roundtripCliScript) = "type MyString =\n  String"
+    ("type MyList = List<String>" |> roundtripCliScript) = "type MyList =\n  List<String>"
+    ("type MyList = List<List<String>>" |> roundtripCliScript) = "type MyList =\n  List<List<String>>"
```

Comment:
TODO: add a test for a List of a custom type

Reply:
Do you want to add this as a comment in the .dark file?

Reply:
I will add the test before the merge, I am just in a different branch currently